$OpenBSD: patch-src_sexyal_drivers_openbsd_cpp,v 1.1 2018/09/08 23:08:39 jeremy Exp $

Adjust for removal of /dev/audio.  Can override default using sound.device
setting in mednafen configuration file.

This is only used when the openbsd driver is forced in the configuration
file, instead of the default sdl driver.

Index: src/sexyal/drivers/openbsd.cpp
--- src/sexyal/drivers/openbsd.cpp.orig
+++ src/sexyal/drivers/openbsd.cpp
@@ -165,7 +165,7 @@ SexyAL_device* SexyALI_OpenBSD_Open(const char* id, Se
 
  AUDIO_INITPAR(&par);
 
- OBSD_TRY(fd = open(id ? id : "/dev/audio", O_WRONLY));
+ OBSD_TRY(fd = open(id ? id : "/dev/audio0", O_WRONLY));
 
  par.bits = SAMPFORMAT_BITS(format->sampformat);
  par.bps = SAMPFORMAT_BYTES(format->sampformat);
